frontend/src/modules/entitieslist/components/EntitiesList.tsx

```diff
@@ -79,26 +76,6 @@ export const EntitiesListBase: React.FC<InternalProps> = (
             isFirstRendered.current = false;
         } else {
             props.dispatch(entities.actions.reset());
-
-            // Scroll to selected entity when entity changes
-            // and when entity list loads for the first time
```

Comment:
We need that code.

Reply:
Ok, I will return it but at production it was not exhibiting that functionality, that's why I assumed it was not necessary.

Reply:
That's surprising. If you move up and down the string list with a keyboard (Alt + Up/Down) or if you open an entity outside the viewport (e.g. https://pontoon.mozilla.org/sl/firefox/all-resources/?string=199689), the list should scroll to the selected entity.

Reply:
> That's surprising. If you move up and down the string list with a keyboard (Alt + Up/Down) or if you open an entity outside the viewport (e.g. https://pontoon.mozilla.org/sl/firefox/all-resources/?string=199689), the list should scroll to the selected entity.

it's still same, it doesn't move. I guess you should try it again or i'll try to figure out how to make it work.

Reply:
Also @mathjazz EntitiesList.test.ts is failing most test because the code written within is based on the old react-infinite-scroller, should i still push my changes if the functionalities works correctly but fails  to pass some test cases?

Reply:
Ok, I'll return that block of code, and also try to check the tests to see what needs to be done

Reply:
Ok, I'll restore the block of code... I deleted my previous message, just discovered the li tag that block of code is trying to select actually exists within the Entity.tsx component, I thought the li tag doesn't exist probably the reason the scrollintoview wasn't working.